core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -51,6 +60,14 @@ static DeflaterOutputStream getEncodingOutputStream(HttpEncodingType encodingTyp
                 }
             case DEFLATE:
                 return new DeflaterOutputStream(out, true);
+            case BR:
+                try {
+                    final Encoder.Parameters parameters = new Encoder.Parameters();
+                    return new BrotliOutputStream(out, parameters.setQuality(4));
```

Comment:
Let's add a comment on why we use 4 as the default level with the link https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html